syntax/psy.vim

```diff
@@ -0,0 +1,37 @@
+" Vim syntax file
+" Language:     Psy
+" Maintainer:   Jean Guyomarc'h <jean.guyomarch@krono-safe.com>
+" Filenames:    *.psy
+" Last Change:  2019 March 28
+" See:          http://www.krono-safe.com/asterios-developer/
+" Version:      This is compatible with Asterios Developer K18 and K19
```

Comment:
Well, only K19 actually, am I right?

When I think about it, we probably should have a separate KhiC syntax file compatible for K18 only: we probably need a first commit that we'll tag as the `K18` release, for our users still using this version _(with .tak, .psy and .khi files)_, then another commit on top of that one that we'll tag as `K19`, with only a Psy syntax file.

...sorry 🤷‍♂️

Reply:
I've split the work into two commits to address both K18 and K19.

Reply:
After discussion, we are back to the original implementation. We will support legacy and various evolutions of the language, just like what is done with C/C++ in upsteam Vim.